apps/app/src/app/shared/ui/tasks-by-status/tasks-by-status.component.ts

```diff
@@ -0,0 +1,177 @@
+import { AsyncPipe, NgFor, NgIf, PercentPipe } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnChanges,
+  SimpleChanges,
+} from '@angular/core';
+import { TaskStatus } from '@armonik.admin.gui/shared/data-access';
+import { Subject, map, tap } from 'rxjs';
```

Comment:
Remove useless imports
```suggestion
import { AsyncPipe, NgFor, NgIf, PercentPipe } from '@angular/common';
import {
  ChangeDetectionStrategy,
  Component,
  Input
} from '@angular/core';
import { TaskStatus } from '@armonik.admin.gui/shared/data-access';
```